pkg/components/wfbackend/sqlite_metadata.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Can you move this file to `pkg/wfengine/sqlite/metadata.go`? This way it's together with the backend file